card/lib/card/loader.rb

```diff
@@ -1,8 +1,18 @@
 # -*- encoding : utf-8 -*-
 
 class Card
+  class << self
+    def config
+      Cardio.config
+    end
+
+    def paths
+      Cardio.paths
+    end
+  end
+
```

Comment:
It's a puzzle from the card-gem branch where class level Card stuff should go when it shouldn't be in card.rb.  I don't think this is bad for these two, but you can move if you want.